v23.1/restoring-backups-across-versions.md

```diff
@@ -16,17 +16,16 @@ Since CockroachDB considers the cluster version when running a backup, this page
 
 ## Support for restoring backups into a higher version
```

Comment:
Could we say "newer" / "older" version instead of "higher" / "lower" version

Reply:
changed to "newer" version